### PR TITLE
Added RatioOfPreferredWeaponPerRound config for configure AWP chance per round.

### PR DESCRIPTION
**Pull request by GL1TCH1337**
If wanted I can make a new Round Type named FullBuyWithAWP and this way we can configure more details like If round is AWP round give more flashes etc.

But this should solve #122 in very basic way.

### CODE REVIEW

File: RetakesAllocatorCore/Config/Configs.cs [hunk 1]
```diff
@@ -218,6 +218,9 @@ public record ConfigData
     public bool EnableNextRoundTypeVoting { get; set; } = false;
     public int NumberOfExtraVipChancesForPreferredWeapon { get; set; } = 1;
     public bool AllowPreferredWeaponForEveryone { get; set; } = false;
+
+    public double PercentageOfPreferredWeaponPerRound { get; set; } = 50;
```
Comment (yonilerner) [inline RetakesAllocatorCore/Config/Configs.cs:222]: Lets call this `ChanceForPreferredWeapon` and set the default to 100

**Comment by GL1TCH1337** [comment]
@yonilerner I made the requested changes. Should i make a new round type and improve this or is this enough for now?

**Comment by yonilerner** [comment]
this change is good as-is, but if you do want to customize more things around the round type, then we should do that instead. However Im concerned adding a new round type is going to be a lot of work. If you want to give it a try though you're more than welcome :)

**Comment by GL1TCH1337** [comment]
Then we can close this PR like this and let the users use this way right now. I will make a new PR when I have more time.

**Comment by yonilerner** [comment]
Sounds good. if we ever add the new round type, we will need to deprecate this new config

**Pull request merged**